docs/src/routes/docs/components/dropdown.svx

```diff
@@ -0,0 +1,106 @@
+---
+name: Dropdown
+relatedComponents:
+  - name: DropdownShell
+    link: ./docs/components/dropdown-shell
+shadowRaised: '0 2px 4px -1px rgba(0, 0, 0, .2), 0 3px 5px 0 rgba(0, 0, 0, .14), 0 1px 10px 0 rgba(0, 0, 0, .12);'
+---
+
+<script>
+  import { DropdownShell, Dropdown, Button } from 'attractions';
+  import Showcase from 'src/containers/docs/showcase.svelte';
+  import ColorPreview from 'src/components/docs/color-preview.svelte';
+  import ShadowPreview from 'src/components/docs/shadow-preview.svelte';
+  import RelatedComponents from 'src/components/docs/related-components.svelte';
+
+</script>
+
+# Dropdown
+
+Toggleable, contextual overlays for displaying additional details or actions. Must be wrapped with a [`DropdownShell`](./docs/components/dropdown-shell) for correct behavior.
+
+<RelatedComponents components={relatedComponents} />
+
+<Showcase>
+  <div slot="showcase" class="padded">
+    <DropdownShell let:toggle>
+      <Button on:click={toggle}>
+        I toggle a dropdown
+      </Button>
+      <Dropdown>
+        <div class="padded">
```

Comment:
This is not enough, they still look crappy. Please, add a SCSS file and give them some padding on all sides.

Reply:
I just copied it from the demo. It was enough there.

Reply:
The demo was inherently crappy :)